contracts-periphery/test/DeployBatchSend.t.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {DeployBatchSend} from "script/DeployBatchSend.s.sol";
+import {Test} from "forge-std/Test.sol";
+import {console2} from "forge-std/console2.sol";
+import {UmbraBatchSend} from "src/UmbraBatchSend.sol";
+import {IUmbra} from "src/interface/IUmbra.sol";
+
+contract DeployBatchSendTest is DeployBatchSend, Test {
+  uint256 constant EXPECTED_TEST_NONCE = 0; // TODO Edit this with the nonce of the
+  address expectedContractAddress = computeCreateAddress(msg.sender, EXPECTED_NONCE);
+  UmbraBatchSend umbraBatchSend = new UmbraBatchSend(IUmbra(UMBRA));
+  bytes batchSendCode = address(umbraBatchSend).code;
+
+  function test_checkNonce() public {
+    assertEq(EXPECTED_NONCE, EXPECTED_TEST_NONCE);
+    assertTrue(expectedContractAddress.code.length == 0);
```

Comment:
Similarly this is testing the contract isn't deployed on forge's vm by default, which seems like a weird thing to test, so maybe you were trying to test something else here?

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/504/commits/ed081a587b4cf48808b00edf7bb85c4b932608ad